File: week2/day4/TheGardenApplication/TheGardenApplication/Tree.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TheGardenApplication
{
    class Tree
    {
        public double WaterAmount { get; set; }
        
        public string Color { get; set; }

        public Tree(string color)
        {
            this.Color = color;
        }

        public void watering(double water)
        {
            this.WaterAmount += (water * 0.4);
        }

        public bool IfNeedWater()
        {
            if (this.WaterAmount < 10)
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }
}

File: DSA-2019/week-02/day-1/tree.py
class Tree():
    def __init__(self, color, water_amount=0):
        self.water_amount = water_amount
        self.color = color

    def need_water(self):
        return self.water_amount < 10

    def absorb_water(self, water):
        self.water_amount += water * 0.4
